src/Domain/Bootstrap.php

```diff
@@ -100,6 +100,12 @@ public function __construct( Container $container ) {
 	protected function init() {
 		$this->register_dependencies();
 		$this->register_payment_methods();
+		$this->load_interactivity_api();
+		add_filter(
+			'woocommerce_blocks_enable_interactivity_api',
+			array( $this, 'has_wp_html_tag_processor' ),
+			999
+		);
```

Comment:
I think this filter should be added before `$this->load_interactivity_api();`. What do you think?
```suggestion
		add_filter(
			'woocommerce_blocks_enable_interactivity_api',
			array( $this, 'has_wp_html_tag_processor' ),
			999
		);
		$this->load_interactivity_api();
```